### PR TITLE
Spelling and grammar cleanup

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Fixes some spelling and grammar from #6984

## Changelog
:cl:
spellcheck: Cowboy clothing now speaks proper-like
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/client/loadout/neck.dm [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	name = "Holocollar"
 	path = /obj/item/clothing/neck/human_petcollar/locked/holocollar
 
-/datum/loadout_item/neck/poncxhocowbot
+/datum/loadout_item/neck/ponchocowboy
 	name = "Green cowboy poncho"
 	path =/obj/item/clothing/toggle/cowboylea
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/loadout/neck.dm]:
```suggestion
	path = /obj/item/clothing/toggle/cowboylea
```

**Comment by theOOZ** [comment]
im posting this here because by now i do not care anymore for any punishment staff can deal to my account. i dont understand why you blocked me or what i said was insulting. i was just lamenting with you the anti climax of leaving a server behind like this is. what the fuck. if those could be my final words.

**Comment by ZephyrTFA** [comment]
> im posting this here because by now i do not care anymore for any punishment staff can deal to my account. i dont understand why you blocked me or what i said was insulting. i was just lamenting with you the anti climax of leaving a server behind like this is. what the fuck. if those could be my final words.

Why the fuck are you putting this on a random PR.
@hugmeorelse

**Comment by theOOZ** [comment]
> > im posting this here because by now i do not care anymore for any punishment staff can deal to my account. i dont understand why you blocked me or what i said was insulting. i was just lamenting with you the anti climax of leaving a server behind like this is. what the fuck. if those could be my final words.
> 
> Why the fuck are you putting this on a random PR.
> @hugmeorelse
 
literally its in the title LOL

**Review by Funce — APPROVED** [review]
Good change.

**Pull request merged**